js/app.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+(function(){
+  angular
+  .module( "songs", [
+    "ui.router",
+    "ngResource"
+  ])
+  .config([
+    "$stateProvider",
+    RouterFunction
+  ]);
+
+  function RouterFunction($stateProvider){
+    $stateProvider
+    .state("index", {
```

Comment:
I would label this as "welcome" or something other than index so there is no confusion with this state and the following one.